replication/binlogsyncer.go

```diff
@@ -57,6 +58,9 @@ type BinlogSyncerConfig struct {
 	ParseTime bool
 
 	LogLevel string
+
+	// RecvBuffer sets the size of the operating system's receive buffer associated with the connection.
+	RecvBuffer int
```

Comment:
The SetReadBuffer argument is `bytes`, so RecvBufferSize makes more sense as the function argument.